scripts/tests/unit/test_populatejsonfiles.py

```diff
@@ -0,0 +1,65 @@
+from requests.exceptions import HTTPError
+import pytest
+import json
+
+import requests
+
+from populatejsonfiles import get, update_json
+
+@pytest.mark.vcr
+def test_get_unit_url_successfully_retrieved():
+    """
+    test response from url request is ok
+    """
+    unit_url = 'http://qudt.org/2.1/vocab/unit'
+    r = get(unit_url)
+    assert r.status_code is requests.codes.ok
+
+@pytest.mark.vcr
+def test_get_quantity_kind_url_successfully_retrieved():
+    """
+    test response from url request is ok
+    """
+    quantity_kind_url = 'http://qudt.org/2.1/vocab/quantitykind'
+    r = get(quantity_kind_url)
+    assert r.status_code is requests.codes.ok
+
+@pytest.mark.vcr
+def test_get_raises_httperror():
+    """
+    test to see if HTTP error is raised
+    """
+    url = 'http://qudt.org/2.1/vocab/this_url_doesnt_exist'
+    with pytest.raises(HTTPError) as excinfo:
+        r = get(url)
+        assert f'Could not get "{url}" with status code {r.status_code}' in str(excinfo)
+
+@pytest.mark.vcr
+def test_update_json():
+    """
+    test update json does not add duplicates
+    """
+    json_path = '/workspaces/reusable-rdf-resources/quantity-kinds.json'
```

Comment:
Can you run these unit tests inside a temporary directory so we don't end up writing over the production files?